snapshot/doc.go

```diff
@@ -2,13 +2,12 @@
 package snapshot offers access to Scoot data.
 Scoot data is largely snapshots (hence the package name).
```

Comment:
Updated a bit...

1) In the Doc, we agreed that Snapshot can be both FSSnapshot and GitCommitSnapshot, so this reflects that.

2) I described that you can use them as the input to a Task, and we intend to store them as the output, but not that that's the exclusive use (you can use Scoot to distribute Snapshots without using CloudExec at all!)